### DISCUSSION TITLE
How to adjust the query time, only query 30 minutes of data

### DISCUSSION THREAD

**Discussion by ZKANGGIT** [apache/skywalking#12305] [General]
![image](https://github.com/apache/skywalking/assets/36831988/2ae5c009-6189-4397-b5d4-bf41931b00c6)

**Comment by wu-sheng** [comment]
What is 30 branches?

**Reply by ZKANGGIT** [reply]
30 minutes

**Reply by wu-sheng** [reply]
Isn't the query range on the right top?

**Reply by ZKANGGIT** [reply]
![image](https://github.com/apache/skywalking/assets/36831988/564a4267-2f72-4a42-8602-d97b9d1b9e94)
There are only the query criteria on the left, but not by time range

**Reply by ZKANGGIT** [reply]
I want to query yesterday's data.

**Reply by wu-sheng** [reply]
<img width="761" alt="image" src="https://github.com/apache/skywalking/assets/5441976/380ef67b-c519-4b28-bc76-7244ac1dd5cc">

Isn't right top clear enough?
